server/src/main/java/com/lunchtime/seed/DatabaseSeed.java

```diff
@@ -47,94 +58,70 @@ public DatabaseSeed(RestaurantRepository restaurantRepository,
 
     @EventListener
     public void seed(ContextRefreshedEvent event) {
-
-        if (personRepository.count() == 0L) {
-            seedPerson();
-        }
-
-        if (restaurantRepository.count() == 0L) {
-            seedRestaurant();
-        }
-
-        if (feedbackRepository.count() == 0L) {
-            seedFeedback();
-        }
-
+        if (personRepository.count() == 0L) seedPerson();
+        if (restaurantRepository.count() == 0L) seedRestaurant();
+        if (feedbackRepository.count() == 0L) seedFeedback();
         if (eventRepository.count() == 0L) {
             try {
                 seedEvent();
             } catch (URISyntaxException e) {
                 e.printStackTrace();
             }
         }
-
     }
 
     public void seedRestaurant() {
-
-        List<Person> person = personRepository.findAll();
-
         for (Long i = Long.valueOf(0); i < restaurantName.length; i++) {
-
-            Restaurant rest = new Restaurant(restaurantName[i.intValue()],
+            Restaurant rest = new Restaurant(
+                restaurantName[i.intValue()],
                 restaurantName[i.intValue()].concat("@gmail.com").toLowerCase(),
                 "Rivne, street ".concat("1" + i.toString()),
                 "www.".concat(restaurantName[i.intValue()]).concat(".ua").toLowerCase(),
-                "Description for restaurant ".concat(restaurantName[i.intValue()]),
-                "12-24", false,
-                i + 1L, person.get(i.intValue()), i.intValue() + 1, cordLongitude[i.intValue()],
-                cordLatitude[i.intValue()], Instant.now(),
-                i + 1L, Instant.now(), i + 1L);
-
+                "A short description about restaurant ".concat(restaurantName[i.intValue()]).concat(", and some other info."),
```

Comment:
What happen with frontend if description will be normal? E.g. 200 - 300 characters?

Reply:
If you create testing data you should add normal (real) information. Not Lorem Ipsum

Reply:
I removed description from cards. I think it's useless in list. I searched through internet and usually there is the most important information to book a restaurant: hours, raiting, address, photo. Because it goes without saying that there are only restaurants. If a user wants to see info, he will open a rest page and will look there full info about it. 

P.S. Nonetheless, my opinion is that not all restaurants will have a big description, maybe some will have few words for example. 

P.S. I think if the main goal is to find were to eat, user usually looks on photos (menu) and working hours, address